### PR TITLE
Update cmake to 3.11.4 on the Docker containers

### PR DESCRIPTION
**Pull request by samuel-rubin**
[skip ci]
Related: #4809
Signed-off-by: Samuel Rubin <samuel.rubin@ibm.com>

### CODE REVIEW

**Comment by samuel-rubin** [comment]
Jenkins build docker s390x ubuntu16

**Comment by samuel-rubin** [comment]
jenkins build docker x86 centos6.9

**Comment by fjeremic** [comment]
Is there a particular issue driving this change? This will increase docker build times significantly.

**Comment by samuel-rubin** [comment]
This is related to #4809. Sorry for not putting that in my commit message. I know that it will increase the build time of the docker images, but if you are using the latest version of the dockerfiles, you can use [these steps](https://github.com/eclipse/openj9/tree/master/buildenv/jenkins/docker-slaves#using-containers-from-a-terminal) to pull the latest from docker hub.

**Comment by samuel-rubin** [comment]
jenkins build docker ppc64le centos7

**Comment by samuel-rubin** [comment]
jenkins build docker ppc64le centos7

**Comment by samuel-rubin** [comment]
jenkins build docker ppc64le centos7

**Comment by samuel-rubin** [comment]
@jdekonin For review

**Comment by dnakamura** [comment]
i believe technically the minimum cmake version required is 3.4 (although we may or may not actually work there)

File: buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile [hunk 1]
```diff
@@ -122,6 +121,16 @@ RUN wget -O - http://cpanmin.us | perl - --self-upgrade \
...
   && cpanm JSON \
   && cpanm XML::Parser
+
+# Install cmake version 3.11.4
+RUN cd /tmp \
+  && wget https://cmake.org/files/v3.11/cmake-3.11.4.tar.gz \
+  && tar xzf cmake-3.11.4.tar.gz \
+  && rm cmake-3.11.4.tar.gz \
+  && cd /tmp/cmake-3.11.4 \
+  && ./configure \
```
Comment (jdekonin) [inline buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile]: Actually, can you investigate turning off the testing that occurs as part of the cmake install?

Comment (samuel-rubin) [reply buildenv/jenkins/docker-slaves/ppc64le/centos7/Dockerfile]: I could not find a way to turn off the testing. @fjeremic Can you merge?

**Pull request merged**